### ISSUE TITLE
ToggleButton may not be calling click handlers

### ISSUE THREAD

**Issue by woutwoot** [stefvanschie/IF#174]
After upgrading to the latest IF version, I'm having issues getting a ToggleButton to call the action lambda supplied.
I do however see the item changing when I click, so the click is being registered. Any idea what this could be?

**Comment by stefvanschie** [comment]
I can't seem to reproduce this. Could you perhaps share your code?

**Comment by woutwoot** [comment]
I have a method that returns a pane, that I then use to add to a GUI. This is what the toggle button related code looks like:
```// Toggle item frame being visible GUI
        ToggleButton toggleHidden = new ToggleButton(0,0,1, 1);
        toggleHidden.setEnabledItem(new GuiItem(Tools.setItemNameAndLore(
                new ItemStack(Material.GLASS_PANE),
                "Item frame display type",
                "Currently hidden"
        ), event -> {
            // Currently hidden, so on click go to visible
            frame.setVisible(true);
        }));
        toggleHidden.setDisabledItem(new GuiItem(Tools.setItemNameAndLore(
                new ItemStack(Material.BROWN_STAINED_GLASS_PANE),
                "Item frame display type",
                "Currently visible"
        ), event -> {
            // Currently visible, so on click go to hidden
            frame.setVisible(false);
        }));
        // If the frame is not visible we need to toggle the button by default
        if(!frame.isVisible()) toggleHidden.toggle();
```
I have since worked around it by emulating the ToggleButton behaviour with StaticPanes.

**Comment by stefvanschie** [comment]
Which version did you update from/to? From version 0.10.2 to 0.10.3?

**Comment by woutwoot** [comment]
I checked my commit history, I did make quite a big version jump. Went from 0.6.1 to 0.10.3
The only change I made because of that was changing Gui into ChestGui, didn't touch anything else.
Apologies for not being able to provide much helpful info :)

**Comment by stefvanschie** [comment]
No problem, I managed to reproduce it.

**Comment by stefvanschie** [comment]
Thank you, this will be fixed in 0.10.5.

**Comment by juanvlr** [comment]
I have the same issue using IF 0.10.5 with this code :
       `ToggleButton tntDestructionButton = new ToggleButton(3, 2, 1, 1, GameConstants.IS_TNT_DESTRUCTIVE);`

    tntDestructionButton.setDisabledItem(new GuiItem(
            new ItemBuilder(Material.TNT)
                    .withName(Component.text("Activer destruction TNT"))
                    .build(),
            event -> {
                event.getWhoClicked().sendMessage(Component.text("La destruction de la tnt est désormais activée"));
                this.configuration.enableTntDestruction();
            }
    ));

    tntDestructionButton.setEnabledItem(new GuiItem(
            new ItemBuilder(Material.TNT)
                    .withName(Component.text("Désactiver destruction TNT"))
                    .build(),
            event -> {
                event.getWhoClicked().sendMessage(Component.text("La destruction de la tnt est désormais désactivée"));
                this.configuration.disableTntDestruction();
            }
    ));
